dockerfile/cargo/Dockerfile

```diff
@@ -187,7 +190,20 @@ RUN ln sh pwd && \
     ln sh env && \
     ln sh tar && \
     ln sh tee && \
-    ln sh du
+    ln sh du && \
+    ln sh date && \
+    ln sh df && \
+    ln sh head && \
+    ln sh md5sum && \
+    ln sh sha1sum && \
+    ln sh sha256sum && \
+    ln sh sha512sum && \
+    ln sh sha3sum && \
+    ln sh stty && \
+    ln sh tail && \
+    ln sh tr && \
+    ln sh which && \
+    ln sh watch
```

Comment:
Might want to alphabetize this list, to make it clear where future entries belong. 

Also have the option to write:

```dockerfile
RUN for b in pwd ls tar cat ...; do ln sh $b; done
```

But it is fine as-is.